lib/fluent/plugin/in_tail.rb

```diff
@@ -49,6 +49,7 @@ def to_s
     end
 
     FILE_PERMISSION = 0644
+    DIR_PERMISSION = 0755
```

Comment:
this value is just copied from https://github.com/fluent/fluentd/blob/9d113029d4550ce576d8825bfa9612aa3e55bff0/lib/fluent/plugin/buf_file.rb#L34